src/screens/SurveyDetail/question.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
 
-import WheelSelect from 'components/WheelSelect';
+import Rating from 'components/Rating';
```

Comment:
@rafayet-monon Should we make this component to be more generic?
For example, if we pass `type` to this component, it will render along with that specific type.
So we don't have to remove some previous codebase like this.
What do you think? 🤔

Reply:
@rafayet-monon @Lahphim Good idea Thumb 👍🏼  we can also pass some optional property to this component for example the choices for wheel selector 🤔

Reply:
@Lahphim @carryall I understand your concern. Keeping the component as it is now is not my plan at all. This component will be generic as the question type will come from the API. These PRs are only to show the implementation of the components. Functionalities will be added in the upcoming PRs. So will the test codes.

Reply:
I already have tasks made for them. For example - https://github.com/rafayet-monon/surveyor/projects/1#card-50002345 where I am adding functionalities and test codes.

Reply:
@rafayet-monon  Okay 👌🏼

Reply:
@rafayet-monon this still be my concern not just only about update the old codebase out and in but also about another PR will dependent to this PR 😅
so you have to fork and chain from this PR, right?

So for my final thought, if you can deal with this, i'll go with you ✌️